Tests/UI.Tests/UnitTests/ControllerTests/PlayedGameControllerTests/SearchHttpPostTests.cs

```diff
@@ -72,14 +72,15 @@ public void ItReturnsTheCorrectViewModelType()
         public void ItReturnsTheCorrectFilterViewModel()
         {
             
-            AutoMocker.Get<IPlayedGameRetriever>().Expect(mock => mock.SearchPlayedGames(null))
+            AutoMocker.Get<IPlayedGameRetriever>().Expect(mock => mock.SearchPlayedGames(null, CurrentUser))
```

Comment:
Since we're ignoring arguments (per the next line), technically you could pass null here and it wouldn't matter. Just an FYI but not that important.